pkg/products/cloudresources/reconciler.go

```diff
@@ -602,3 +617,32 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 
 	return client.Patch(ctx, cfgMap, k8sclient.Merge)
 }
+
+// createSTSARNSecret create the STS arn secret - should be already validated in preflight checks
+func (r *Reconciler) createSTSARNSecret(ctx context.Context, client k8sclient.Client, operatorNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
no strong opinion but I would rather see this function under resources > sts or some other helper file